java8/Java8.g4

```diff
@@ -348,7 +348,8 @@ fieldModifier
 	;
 
 variableDeclaratorList
-	:	variableDeclarator (',' variableDeclarator)*
+	:	variableDeclarator
+	|	variableDeclaratorList ',' variableDeclarator
```

Comment:
I don't understand clearly, but I think the number of ATN states depend on rule number. You can do your own research and compare performance results.